include/sys/fs/zfs.h

```diff
@@ -1346,7 +1363,7 @@ typedef enum {
  */
 typedef enum zfs_ioc {
 	/*
-	 * Core features - 81/128 numbers reserved.
+	 * Core features - 88/128 numbers reserved.
```

Comment:
I assume the bump from 81 -> 88 is correct?

Reply:
The comment has not been updated in some time. I think I counted correctly :)